myp5.js

```diff
@@ -51,44 +47,53 @@ function draw() {
   }  
 }
 
-function lyrics(x, y) {
+function loadLyricsOnce() {
```

Comment:
This is the function we call in the successful ajax response to create all the lyric objects with the `lyrics` array